src/layout/Tabbed.js

```diff
@@ -69,6 +69,14 @@
         HTMLWidget.prototype.update.apply(this, arguments);
         var context = this;
 
+        if (this._tabContainer) {this._tabContainer.remove();}
+        if (this.tabsOnBottom()) {
+            this._tabContainer = element.append("div");
```

Comment:
Its technically not necessary, I only classed it below because the css is different when its on the bottom.